docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-overview-con.adoc

```diff
@@ -0,0 +1,41 @@
+[id='case-management-overview-con']
+= Case management
+
+Case management is an extension of Business Process Management (BPM) that allows you to handle dynamic and ad hoc activities. 
+
+BPM is a management practice for automating tasks that are repeatable and have a common pattern, with a focus on optimization by perfecting a process. Business processes are usually modeled with clearly defined paths leading to a business goal, often with a starting point that is structurally connected to build an end to end flow of work and data. This requires a lot of predictability, usually based on mass-production principles. However, many applications in the real world cannot be described completely from start to finish (including all possible paths, deviations, and exceptions). Moreover, using a process oriented approach in certain cases may lead to complex solutions that are hard to maintain. 
```

Comment:
Minor and up to you:
Might consider changing: However, many applications in the real world cannot be described..
To: However, many real world applications cannot be described..